tests/templates/test_subroutines/test_basis_rotation.py

```diff
@@ -24,13 +24,13 @@
 @pytest.mark.xfail  # to be fixed by shortcut story 49160
 def test_standard_validity():
     """Run standard tests of operation validity."""
-    weights = np.array(
+    unitary_matrix = np.array(
         [
             [-0.618452, -0.68369054 - 0.38740723j],
             [-0.78582258, 0.53807284 + 0.30489424j],
         ]
     )
-    op = qml.BasisRotation(wires=range(2), unitary_matrix=weights)
+    op = qml.BasisRotation(unitary_matrix=unitary_matrix, wires=range(2))
```

Comment:
This is the suggestion I meant:
```suggestion
    op = qml.BasisRotation(unitary_matrix, wires=range(2))
```

Reply:
Oh, now I see what you mean. Thanks!